vendor/engines/secure_rooms/spec/controllers/secure_rooms_api/events_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+RSpec.describe SecureRoomsApi::EventsController do
+  before do
```

Comment:
Might want to extract this out into a separate method at some point, but I think holding off until we have one more instance of it would be a-okay!